yarn.lock

```diff
@@ -0,0 +1,8254 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Can we remove this? v2.0.0 has yarn removed in favor of `package-lock.json` (node >8) since apparently it is faster or at least equivalent. Node 9 has been added to travis to confirm that builds work and tests pass correctly with that version too.